properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -872,20 +872,26 @@ making sure that you maintain a proper JSON format.
 						"propertyKeyword": "constrDescription",
 						"propertyTooltip": "description",
 						"propertyType": "details",
-						"template": "textarea"
+						"template": "codeEditor",
```

Comment:
```
"template": "textarea",
"markdown": true
```

Reply:
Code editor has "HTML" preview as well, so the issue is resolved